src/aspire/estimation/covar2d.py

```diff
@@ -20,13 +22,13 @@ class RotCov2D:
     J. Struct. Biol. 195, 27-81 (2016). DOI: 10.1016/j.jsb.2016.04.013
     """
 
-    def __init__(self, basis, as_type='single'):
+    def __init__(self, basis):
         """
         constructor of an object for 2D covariance analysis
         """
         self.basis = basis
         ensure(basis.ndim == 2, 'Only two-dimensional basis functions are needed.')
-        self.as_type = as_type
+        self.dtype = 'double'
```

Comment:
Why do we need to store this?

Reply:
As above, it is used to make consistent with other classes. Removed in the new commit # 4c22159.